package.json

```diff
@@ -2,8 +2,8 @@
   "name": "@firebolt-js/sdk",
   "version": "0.6.1-beta.2",
   "description": "The Firebolt JS SDK",
-  "main": "./dist/firebolt.mjs",
-  "types": "./dist/firebolt.d.ts",
+  "main": "./dist/lib/firebolt.mjs",
```

Comment:
I see you're putting firebolt.mjs into dist/lib/ (not just dist/)... This seems fine, but just curious why. Will there be anything in dist/ other than the lib/ directory?